clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -60,6 +60,9 @@ def err_incompatible_fp_eval_method_options : Error<
     "option 'ffp-eval-method' cannot be used with option "
     "%select{'fapprox-func'|'mreassociate'|'freciprocal'}0">;
 
+def err_incompatible_fp_accuracy_options : Error<
+    "option 'ffp-accuracy' cannot be used with option 'fmath-errno'">;
```

Comment:
Don't we already have a more general "option X cannot be used with option Y" diagnostic instead of hard-coding the strings?

Reply:
There's `err_drv_incompatible_options` in the driver if you want to move it to DiagnosticCommonKinds.td and use it, but I don't insist (I thought we had a general one already in use, but we don't -- that one is only used once and it's got issues with diagnostic quality that I'm going to fix upstream).